mscore/musescore.cpp

```diff
@@ -2990,6 +2993,19 @@ void MuseScore::clipboardChanged()
       }
 
 //---------------------------------------------------------
+//   inputMethodLocaleChanged
+//---------------------------------------------------------
+
+void MuseScore::inputMethodLocaleChanged()
+      {
+      QInputMethod* im = QGuiApplication::inputMethod();
+      _qInputMethodScript = im->locale().script();
+      qDebug("new input method locale: %d", _qInputMethodScript);
+      if (_qInputMethodScript == QLocale::JapaneseScript)
+            qDebug("Might need to handle japanese IME special chars");
+      }
```

Comment:
I just realized now that _qInputMethodScript is going to be left uninitialized until IME changes.  This might be problemnatic...maybe I should have not even store _qInputMethodScript but rather just query the inputMethod only when needed.